commands/keplr.js

```diff
@@ -319,10 +359,22 @@ const keplr = {
     );
     const innerTexts = await parentElement.allInnerTexts();
     const textArray = innerTexts[0].split('\n');
-    const tokenValue = Number(textArray[3]);
 
-    await playwright.switchToCypressWindow();
+    let numberString;
+    for (const element of textArray) {
+      const elementWithNoCommas = element.replace(/,/g, '');
+      if (!isNaN(parseFloat(elementWithNoCommas))) {
+        numberString = elementWithNoCommas;
+        break;
+      }
+    }
 
+    if (!numberString) {
+      throw new Error(`Token value for ${tokenName} not found`);
+    }
+
+    let tokenValue = parseFloat(numberString);
+    await playwright.switchToCypressWindow();
```

Comment:
i dont think we need to make it this complicated:
```
const tokenValue = textArray[3];
const parsedTokenValue = Number(element.replace(/,/g, ''));
await playwright.switchToCypressWindow();
```
should suffice

Reply:
Based on the string we are grabbing and extracting the number, if we analyze it, it just contains one number. 
This is working and giving accurate results. For the new UI, we are not sure what it will contain or what changes Keplr will introduce. We don't control the UI Keplr and in that sense, both approaches are bad then.